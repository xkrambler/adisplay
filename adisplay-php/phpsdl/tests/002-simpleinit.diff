001+ int(-1)
002+ int(16385)
001- int(0)
002- int(29233)