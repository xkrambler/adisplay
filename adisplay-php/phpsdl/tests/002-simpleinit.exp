int(0)
int(29233)
NULL
Done