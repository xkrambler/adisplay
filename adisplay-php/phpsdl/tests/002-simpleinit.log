
---- EXPECTED OUTPUT
int(0)
int(29233)
NULL
Done
---- ACTUAL OUTPUT
int(-1)
int(16385)
NULL
Done
---- FAILED
