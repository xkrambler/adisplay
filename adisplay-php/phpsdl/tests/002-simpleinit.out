int(-1)
int(16385)
NULL
Done